docs/QTIP.md

```diff
@@ -0,0 +1,30 @@
+# QTIP
+
+## What?
+
+QTIP is an MsQuic specific concept that allows MsQuic to exercise the QUIC protocol over XDP using TCP packets.
+
+To be specific, QTIP adds a TCP header to the QUIC packet instead of a UDP header. It doesn't follow the TCP protocol after the initial handshake: headers are generated with good enough data to be compatible with most networks and leverage TCP specific optimizations, but are not used to operate the protocol.
+
+You can think of it as disguising QUIC packets as TCP packets!
+
+## Why?
+
+Certain hardware / networks / cloud environments are optimized for TCP traffic. Instead of pushing for those environments to also optimize for UDP, QTIP allows us to leverage existing hardware with MsQuic.
+
+## How?
+
+To use QTIP, you first must be running XDP. The setting is ignored if XDP is not enabled.
+
+QTIPEnabled in [QUIC_SETTINGS](./api/QUIC_SETTINGS.md) controls whether client connections will send/recv packets over QTIP xor UDP and whether listeners will optionally accept QTIP traffic.
```

Comment:
I think this can be simplified a little (and remove the potentially confusing xor) to
"QTIPEnabled in [QUIC_SETTINGS](./api/QUIC_SETTINGS.md) controls whether client connections exclusively use QTIP, and whether listeners will optionally accept QTIP traffic."

Reply:
Done